### PR TITLE
upload file listing

### PR DESCRIPTION
**Pull request by mxcl**

### CODE REVIEW

File: .github/workflows/pkg-platform.yml [hunk 1]
```diff
@@ -135,6 +135,13 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: us-east-1
 
+      - name: mock `aws`
+        if: inputs.dry-run
+        run: |
+          echo '#!/bin/sh' > /usr/local/bin/aws
+          echo 'echo "$@"' >> /usr/local/bin/aws
+          chmod +x /usr/local/bin/aws
+
```
Comment (jhheider) [inline .github/workflows/pkg-platform.yml]: won't have to worry about path issues (or polluting our builders!!!) if you just set `AWS` in github_env, and replace lower calls with `$AWS`.

File: .github/workflows/pkg-platform.yml [hunk 2]
```diff
@@ -125,6 +125,7 @@ jobs:
         compression: [xz, gz]
     runs-on: ubuntu-latest
     env:
+      AWS: ${{ inputs.dry-run && 'aws' || 'echo '}}
```
Comment (jhheider) [inline .github/workflows/pkg-platform.yml]: ooh, very slick.

Comment (jhheider) [reply .github/workflows/pkg-platform.yml]: if it had worked. hm. i wonder if it's needs the `== true`. might explain why the other `input.dry-run &&` things look like they're not working.

File: .github/workflows/pkg-platform.yml (hunk 1 above)
Comment (mxcl) [reply .github/workflows/pkg-platform.yml]: notably this only ran on `ubuntu-latest` so was probs safe

File: .github/workflows/pkg-platform.yml (hunk 2 above)
Comment (jhheider) [reply .github/workflows/pkg-platform.yml]: oh. ha. just logic reversed. i'll fix it.

**Pull request merged**